packages/plugin-react-native-app-state-breadcrumbs/test/app-state.test.ts

```diff
@@ -36,53 +40,24 @@ describe('plugin: react native app state breadcrumbs', () => {
   })
 
   it('should not be enabled when enabledBreadcrumbTypes=null', () => {
-    let _cb
-    const AppState = {
-      addEventListener: (type, fn) => {
-        _cb = fn
-      }
-    }
-    const plugin = proxyquire('../', {
-      'react-native': { AppState }
-    })
-
+    // @ts-ignore
```

Comment:
Apparently `null` is not a valid type for `enabledBreadcrumbTypes` but setting to `undefined` causes the test to fail. Needs further investigation.

Reply:
the types are wrong – according to the spec `null` is an allowed value. Let's not hold up this PR because of it. I'll make a note to fix it elsewhere.